src/Connection.c

```diff
@@ -593,13 +593,13 @@ static int Connection_Init(udt_Connection *self, PyObject *args,
     if (dpiContext_initConnCreateParams(g_DpiContext, &dpiCreateParams) < 0)
         return Error_RaiseAndReturnInt();
     if (!PyArg_ParseTupleAndKeywords(args, keywordArgs,
-            "|OOOikO!OOOiOssOOOO", keywordList, &usernameObj, &passwordObj,
+            "|OOOikO!OOOiOssOOOOI", keywordList, &usernameObj, &passwordObj,
```

Comment:
I think `I` makes more sense here:

https://docs.python.org/2/c-api/arg.html

Upper vs lower case is unsigned/signed. This is a size and it shouldn't ever be negative. In the Oracle example it's a `ub4` which is apparently an unsigned int.

http://docs.oracle.com/database/122/LNOCI/lobs-and-bfile-operations.htm#GUID-3F7FEAFF-F4EF-42CC-8F95-598D227C5D83__BGBGCGCB

http://www.mit.edu/afs.new/sipb/project/oracle-client/sdk/include/oratypes.h